.github/linter-rules/.yaml-lint-config.yaml

```diff
@@ -63,4 +63,4 @@ rules:
   octal-values: disable
   quoted-strings: disable
   trailing-spaces: enable
-  truthy: disable
+  truthy: disable
```

Comment:
looks like an end of file newline was removed?

Reply:
@alaincormier-ssc  - I restored the original file. Please mark as resolved if satisified.